llama_index/response/notebook_utils.py

```diff
@@ -39,17 +39,26 @@ def display_extra_info(extra_info: Dict[str, Any]) -> None:
     display(extra_info)
 
 
-def display_response(response: Response, source_length: int = 100) -> None:
+def display_response(
+    response: Response,
+    source_length: int = 100,
+    show_source: bool = False,
+    show_extra_info: bool = False,
```

Comment:
is this being used in the nb?

Reply:
Yea, just so that prints have text wrapping by default.